components/centraldashboard/public/components/registration-page_test.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
+import '@polymer/test-fixture/test-fixture';
+import 'jasmine-ajax';
+import {mockIronAjax, yeildForRequests} from '../ajax_test_helper';
+import {flush} from '@polymer/polymer/lib/utils/flush.js';
+
+import './dashboard-view';
+
+const FIXTURE_ID = 'registration-page-fixture';
+const MU_VIEW_SELECTOR_ID = 'test-registration-page';
+const TEMPLATE = `
+<test-fixture id="${FIXTURE_ID}">
+  <template>
+    <registration-page id="${MU_VIEW_SELECTOR_ID}"></registration-page>
+  </template>
+</test-fixture>
+`;
+const TEST_USER = 'test@kubeflow.org';
+const sleep = (t) => new Promise((res) => setTimeout(res, t));
```

Comment:
Do you want to use the helper functions you defined in the ajax_test_helper module?

Reply:
Yup, removed these lines